App/Libraries/Ldap.php

```diff
@@ -70,8 +70,8 @@ private function getInfosUser($nom)
 
     public function getEmailUser($login)
     {
-        $filter = "(&(" . $this->configuration['attrLogin'] . "=" . $login . ")
-                    (" . $this->configuration['attrFiltre'] . "=" . $this->configuration['filtre'] . "))";
+        $filter = "(&(" . $this->configuration['attrLogin'] . "=" . $login . ")(" . 
```

Comment:
Codacy found an issue: [Whitespace found at end of line](https://app.codacy.com/manual/Libertempo/web/pullRequest?prid=6401132)

Reply:
Commit edited.